torchbenchmark/models/resnet50/__init__.py

```diff
@@ -1,29 +1,37 @@
 
-# Generated by gen_torchvision_benchmark.py
 import torch
 import torch.optim as optim
 import torchvision.models as models
 from ...util.model import BenchmarkModel
 from torchbenchmark.tasks import COMPUTER_VISION
 
-#######################################################
-#
-#       DO NOT MODIFY THESE FILES DIRECTLY!!!
```

Comment:
I'm a bit scared by this though. What's the purpose of the comment in the first place?

Reply:
These comments were added because this file was generated by another script (https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/gen_torchvision_benchmarks.py). The idea was that all torchvision models should share the same code template.
Later we found we can't really do that, because each of them needs to be customized (e.g. batch size, jit, and like this, fx2trt). So we are removing this alert.